src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -0,0 +1,271 @@
+
+
+@main.command(
+    short_help="Submit jobs to an HPC cluster using PyHPS package.",
+    help="""Submit jobs to an HPC cluster using PyHPS package.""",
+)
+@click.argument("main_file")
+@click.option(
+    "--name",
+    default=None,
+    type=str,
+    help="""Name of the PyHPS project to be created.""",
+)
+@click.option(
+    "--url",
+    default=None,
+    type=str,
+    help="""URL where the HPS cluster is deployed. For example: "https://myserver:3000/hps" """,
+)
+@click.option(
+    "--user", default=None, type=str, help="Username to login into the HPC cluster."
+)
+@click.option(
+    "--password",
+    default=None,
+    type=str,
+    help="Password used to login into the HPC cluster.",
+)
+@click.option(
+    "--python",
+    default=None,
+    type=str,
+    help="""Set python version to be used to create the virtual environment and
+run the python file. By default it uses python3 (default in cluster).""",
+)
+@click.option(
+    "--output_files",
+    default=None,
+    type=str,
+    help="""Set the output files to be monitored.""",
+)
+@click.option(
+    "--shell_file",
+    default=None,
+    type=str,
+    help="""If desired, you can provide a shell script to execute instead of
+the python file. You can call your python file from it if you wish. By default,
+it is not used.""",
+)
+@click.option(
+    "--requirements_file",
+    default=None,
+    type=str,
+    help="""If provided, the created virtual environment is installed with the
+libraries specified in this file. If not, the activated virtual environment is
+cloned through a temporary 'pip list' file. If you are using editable package,
+it is recommended you attach your own requirement file using ``pip freeze`` """,
+)
+@click.option(
+    "--extra_files",
+    default=None,
+    type=str,
+    help="""To upload extra files which can be called from your main python file
+(or from the shell file).""",
+)
+@click.option(
+    "--config_file",
+    default=None,
+    type=str,
+    help="""To load job configuration from a file.""",
+)
+@click.option(
+    "--num_cores",
+    default=None,
+    type=str,
+    help="""Set the amount of CPU cores reserved for the job. By default it is 1 CPU.""",
+)
+@click.option(
+    "--memory",
+    default=None,
+    type=str,
+    help="""Set the amount of memory RAM in MB reserved for the job. By default it is 100 MB.""",
+)
+@click.option(
+    "--disk_space",
+    default=None,
+    type=str,
+    help="""Set the amount of hard drive space in MB reserved for the job. By default it is 100 MB.""",
+)
+@click.option(
+    "--exclusive",
+    default=None,
+    type=str,
+    is_flag=False,
+    flag_value=True,
+    help="""Set the job to run in a machine exclusively, without sharing it with other jobs. By default it is False""",
+)
+@click.option(
+    "--max_execution_time",
+    default=None,
+    type=str,
+    help="""Set the maximum execution time for the job. By default it is zero (unlimited).""",
```

Comment:
```suggestion
    help="""Maximum execution time for the job. The default is zero (unlimited).""",
```
WIth a string time, is the default ``zero (unlimited)`` or either of these values. It's unclear.

Reply:
Good catch!